packages/frameworks/frameworks.json

```diff
@@ -168,6 +168,37 @@
       }
     }
   },
+  {
+    "name": "Dojo",
+    "slug": "dojo",
+    "demo": "https://dojo.now-examples.now.sh",
```

Comment:
Is deploying to `https://dojo.now-examples.now.sh` something that I need to do or is it something that happens automatically?

Reply:
I think we do it manually, right @msweeneydev ?